### DISCUSSION TITLE
MIB/VAG - acronyms explained - HELP NEEDED!

### DISCUSSION THREAD

**Discussion by sgrobler** [Mr-MIBonk/M.I.B._More-Incredible-Bash#136] [General]
As a newbie I struggled with some of the acronyms used - might be helpful for others (including me) to have a list.

Acroynm | Description
-- | --
FW | Firmware
MU | Main Unit
VAG | Volkswagen AG
MIB | Modularen Infotainmentbaukasten
MIB2 High   MIB2.5 High | Skoda:   Columbus  </br> VW: Discover Pro  </br> Seat: Navi System Plus  </br> Audi: MMI4G/MMI Plus </br>Porsche:PCM4/4.1</br>  all manufactured by Harman 
MMI | Multi Media Interface (Audi   MIB)
FEC | Function Enabling Code
SWaP | SoftWare as a Product
RCC | Radio & Car Control   Unit
MMX | Multi-Media Extension   Board
CP | Component Protection
SWDL | Software Download
full SWDL | 
ODIS-S | Offboard Diagnostic Information System   - Service
ODIS-E | Offboard Diagnostic Information System   - Engineering
OBDeleven | 
ELM327 | 
CarScanner | 
VCP | 
VCDS | 
Geko | Geheimnis   und Komponentenschutz
VIN | Vehicle Identification   Number
VCRN | ?
PQ | 
ZR | 
Patch | 
SW train | 
full SWDL | 
Toolbox | 
M.I.B | 
AIO | All-in-One - used for custom firmware updates which come with latest software components (like GraceNote and RadioStation Database) and are pre patched.
AA | Android Auto
CP | (Apple) CarPlay
GEM | Green Engineering Menu - a hidden menu which has to be enabled by coding  - normally via OBD changing adaptions and enabling [developer mode](https://www.remkoweijnen.nl/blog/2016/05/17/enable-developer-mode-on-vw-discover-pro-with-vcds/) - will be enabled via software during M.I.B install
MHI | 
MHI2 | 
MHI2Q | 
MMI | 
PCM | 
VIM | Video in Motion - normally mentioned as people want to remove speed limitations on playing back videos in car
FEC | 
FW | 
SVM | Software Version Management
VC | 

Also check: www.mibwiki.one

**Comment by harman-f** [comment]
@sgrobler 
Nice idea!
I added a bit to the list, please fill out as much as you can!
Also supportings links to external content are fine, if it helps to explain in more detail.

**Comment by Direcka** [comment]
@sgrobler 
Abrev - Full - Description
PQ - Passagier Quer - Definition of type of vehicle platform
M.I.B. - Modular Infotainment Baukasten - central unit of infotainment. Its used also as More Incredible Bash in wiki
ZR - Zentral Rechner - in translation it is computation unit, may be any electrical unit in car
FW - Firmware

**Comment by mrfixpl** [comment]
full SWDL - software download (firmware update) that overwrites all content instead (allows downgrades and overwriting modified files)

PCM - Porsche Communication Management

FEC - Feature Enabling Code / Function Enabling Code

VC - Virtual Cockpit / AID - Active Info Display

MHI / MHIG - MIB1 High
MHI2 - MIB2 High
MHI2Q - MIB2 High with QualComm chip

SW Train - firmware signature contains information about main unit generation and tier, destination region, vehicle brand (and model or platform), firmware release type (production, beta, patch...), actual version of hardware and software, and sometimes hardware manufacturer. It can look like this: MST2_EU_VW_ZR_P0480T.
MST2 - MIB2 Standard
EU - Europe region
VW_ZR - Volkswagen vehicle with separate main unit
P - production firmware
0480 - software version
T - hardware manufactured by Technisat
Firmware name is so long and looks like it has railcars. So let's call it train.

I've collected couple more things over here: http://wiki.mib-helper.com/index.php?title=Acronyms_Dictionary
